openstack/intel-worker/tasks/update-chrony-configuration.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: Update the Chrony config
+  hosts: all
+  vars_files:
+    - ../vars/main.yml
+
+  tasks:
+
+  - name: Generate the subnet list
+    shell:
+      cmd: |
+        # Create the subnets.yml
+        cat << EOF > subnets.yml
+        ---
+        subnets:
+        EOF
+        # Populate data in subnets.yml
+        for SUBNET in $(ip r | grep -v default | awk '{print $1}')
+        do
+        cat << EOF >> subnets.yml
+          - { subnet: '${SUBNET}'}
+        EOF
+        done
```

Comment:
```suggestion
  - name: Generate the subnet list
    shell: ip r | grep -v default | awk '{print \$1}'
    register: subnet_list

  - name: Print the subnets
    ansible.builtin.debug:
      var: subnet_list
```

Reply:
shell: ip r | grep -v default | awk '{print \$1}' is failing with below mentioned error. [Unwanted  \ for {print $1} is causing issue. ] 
We should use shell: ip r | grep -v default | awk '{print $1}'

fatal: [localhost]: FAILED! => {"changed": true, "cmd": "ip r | grep -v default | awk '{print /$1}'", "delta": "0:00:00.005722", "end": "2024-02-29 06:25:23.592387", "msg": "non-zero return code", "rc": 1, "start": "2024-02-29 06:25:23.586665", "stderr": "awk: cmd. line:1: {print /$1}\nawk: cmd. line:1:         ^ unterminated regexp\nawk: cmd. line:1: {print /$1}\nawk: cmd. line:1:            ^ unexpected newline or end of string", "stderr_lines": ["awk: cmd. line:1: {print /$1}"

Reply:
I think in the above command the slash is facing the wrong way